app/models/form/lettings/questions/address_line1.rb

```diff
@@ -7,6 +7,7 @@ def initialize(id, hsh, page)
     @type = "text"
     @plain_label = true
     @check_answer_label = "Q12 - Address"
+    @do_not_clear = true
```

Comment:
Is there a more descriptive name than this? It's not self-evident what it does to me!